modules/serverless-logic-building-deploying-workflow-preview-mode.adoc

```diff
@@ -112,5 +69,13 @@ Ensure that the pod corresponding to your workflow is running.
 +
 [source,terminal]
 ----
-$ oc logs pod/<pod-name> -n workflows
+$ oc logs pod/<pod-name> -n <namespace>
+----
+
+.Cleanup
+. To cleanup the deployment, delete the projects resources running the following command:
```

Comment:
Either way though, If this stays as a single-step procedure, use an unordered list (per [guidelines](https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs)) and use "by running":
```suggestion
* To cleanup the deployment, delete the projects resources by running the following command:
```